src/adzerk/boot_reload/reload.cljs

```diff
@@ -21,8 +21,8 @@
 (defn- changed-href? [href-or-uri changed]
```

Comment:
lol, I can add the `docstring` yeah, about the name...man that's the most difficult part of this patch!